Mathlib/Data/Bitvec/Core.lean

```diff
@@ -361,17 +391,18 @@ end Conversion
 
 
 private def repr {n : Nat} : Bitvec n → String
-  | ⟨bs, p⟩ => "0b" ++ (bs.map fun b : Bool => if b then '1' else '0').asString
-#align bitvec.repr bitvec.repr
+  | ⟨bs, _⟩ => "0b" ++ (bs.map fun b : Bool => if b then '1' else '0').asString
+--#align bitvec.repr Bitvec.repr
+--porting note: Removed align for a private declaration.
+--Reference: https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/align.20private.20decls/near/317107099
```

Comment:
I suspect we need no note here, especially since the plan is to make `mathport` do this for us.